index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Microverse Portfolio</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+</head>
+<body>
+  <header>
+    <a id="logo" class="color-blue" href="#">
+      My Logo
+    </a>
+    <nav>
+      <span class="material-symbols-outlined color-blue">
+        menu
+      </span>
+    </nav>
+  </header>
+  <section id="intro-section">
+    <h1 class="color-dark-blue">
+      I'm Precious
+      <br>
+      Glad to see
+      <br>
+      you!
+    </h1>
+    <p>
+      I&apos;m a software developer! I can help you
+      <br>
+      build a product, feature or website. Look
+      <br>
+      through some of my work and
+      <br>
+      experience! If you like what you see and
+      <br>
+      have a project you need coded, don&apos;t
+      <br>
+      hestiate to contact me.
+    </p>
+    <span id="connect-text" class="color-blue">LET'S CONNECT</span>
+    <ul id="social-media">
+      <li>
+        <img src="images/twitter.svg" alt="twitter">
+      </li>
+      <li>
+        <img src="images/linkedin.svg" alt="linkedin">
+      </li>
+      <li>
+        <img src="images/m.svg" alt="m">
+      </li>
+      <li>
+        <img src="images/github.svg" alt="github">
+      </li>
+      <li>
+        <img src="images/rabbit.svg" alt="rabbit">
+      </li>
+    </ul>
```

Comment:
- [ ] Great job using `<ul>/<li>`  tags for for wrapping social media. However, please make sure that every link/icon is rendered using the anchor `<a>` tag as suggested by previous reviewer 🔗
If you see the websites, they have social media icons with links that are directed to the respective website. The point here is to do the same.  I know you can do that 💪